src/components/site-header/demo/data--ec.js

```diff
@@ -22,6 +22,27 @@ module.exports = {
     label: 'Log out',
     href: exampleLink,
   },
+  custom_action: {
+    link: {
+      path: exampleLink,
+      label: 'Custom label',
+      icon: 'brochure',
```

Comment:
I just noticed, but the data structure for the icon is not correct; it should be defined as an object outside of the link

Reply:
true, i moved it outside now of link